data/performance.json

```diff
@@ -204,6 +205,7 @@
     "National Institute for Biological Standards and Control": 1,
     "National Institute of Health and Care Excellence": 1,
     "National Museums NI": 1,
```

Comment:
"National Museums NI": 1,
    "National Museums Northern Ireland": 1,

These are the same organisation, so the count should be 2.